run_dir/design/index.html

```diff
@@ -54,7 +54,7 @@ <h3><a href="https://ngi-dashboards.scilifelab.se/dbe/"><i class="glyphicon glyp
 		<h3>Server Status</h3>
```

Comment:
And finally I would throw in a `class="mt-5"` on the row div above.

Reply:
Done!